docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -185,12 +186,12 @@ By default, Solaris 10 and earlier run a non-POSIX shell which does not correctl
 tmp directory Ansible uses ( :file:`~/.ansible/tmp`). If you see module failures on Solaris machines, this
 is likely the problem. There are several workarounds:
 
-* You can set :ref:`remote_tmp` to a path that will expand correctly with the Solaris shell.  For
+* You can set ``remote_tmp`` to a path that will expand correctly with the Solaris shell.  For
```

Comment:
I think this should link to remote_tmp documentation.  With bcoca's config rewrite, it's now documented in each shell plugin... For instance, in plugins/shell/sh.rst

Reply:
I haven't figured out how to do this - I don't think there are any anchors in the raw html of the individual plugin docs. And I'm not sure which of the shell plugins this particular reference should point to. Future work?

Reply:
The canonical entry is the major problem to me. We could (and probably should) add namespaced anchors for each config entry to the template files but it still won't help us decide which one is the correct one to link to from other documentation :-(

But yeah, with so many open questions, I think we either have to defer it to the future or include it as something that gets a documentation stub for now (like we figured on doing for the :py:meth: links....  What do you think?